build/build-options.js

```diff
@@ -0,0 +1,106 @@
+require("dotenv").config();
+
+const {
+  FLAW_LEVELS,
+  DEFAULT_FLAW_LEVELS,
+  VALID_FLAW_CHECKS,
+  FILES,
+  FOLDERSEARCH,
+  NO_PROGRESSBAR,
+} = require("./constants");
+
+module.exports = {
+  flawLevels: parseFlawLevels(DEFAULT_FLAW_LEVELS),
+  files: parseFiles(FILES),
+  foldersearch: parseFoldersearch(FOLDERSEARCH),
+  noProgressbar: NO_PROGRESSBAR,
```

Comment:
Why the re-export?

Reply:
I don't get it.

Reply:
Yeah actually nevermind, I was wondering why `NO_PROGRESSBAR` isn't imported directly where it's needed. But then again, this current way makes it easier to replace with CLI options in the future, so I do like it more. Nevermind me 🙊

Reply:
I want whoever uses `require('./build-options')` to refer to it as `noProgressbar`.